### ISSUE TITLE
Night video while streaming looks fine, but when saved as movie looks too dark

### ISSUE THREAD

**Issue by kevkid** [Motion-Project/motion#747]
1. version [x.y.z, hash, other]: server: 4.0, motioneyeos 4.1.1
2. installed as a package or compiled from sources [deb, rpm, git, other]: apt install, part of motioneyeos
3. standalone or part of third party [motion, MotionEyeOS, other]: Both My server has motioneye installed from apt, my raspberrypi zeros which have the camera attached have motioneyeos
4. video stream source [V4L (card or USB), net cam (mjpeg, rtsp, other), mmal]: net cam
5. hardware [x86, ARM, other]: arm
6. operating system [16.04, Stretch, etc, FreeBSD, other]: server has ubuntu 16.04 and raspberry pi zeros have motioneyeos
Hi,
I have been having an issue where when I am recording at night my movie files seems to be darker than the stream. I am not exactly sure why, I have set the quality of the movie to 100 and it still shows up really dark.

Here is an example of what I am talking about. The left image is a screenshot of the movie I am playing via the web interface (looked the same in VLC, very dark). The right is a screenshot of the stream as it is happening. 
![](https://i.imgur.com/m7h5GLm.png)

We can see that clearly there is a difference in brightness. Is there anything I should be looking at?
Here is the hardware I am using:

- Makerfocus Raspberry Pi Zero W Camera Night Vision Webcam 2 Infrared IR LED Light for Raspberry Pi Zero and Zero W [found here](https://www.amazon.com/gp/product/B073HYJDCM/ref=oh_aui_detailpage_o02_s00?ie=UTF8&psc=1)

- Raspberry Pi Zero W [found here](https://www.amazon.com/gp/product/B06XJQV162/ref=oh_aui_detailpage_o02_s01?ie=UTF8&psc=1)

**Comment by tosiara** [comment]
Can you try to reproduce one again and save stream video using ffmpeg command:
`ffmpeg -i http://motion:8081/ -an -vcodec copy file.avi`
And share again two videos - one recorded, and another - saved stream
Also would be nice to see your motion.conf, which codec are you using for movies?

**Comment by kevkid** [comment]
> Can you try to reproduce onсe again and save stream video using ffmpeg command:
ffmpeg -i http://motion:8081/ -an -vcodec copy file.avi
And share again two videos - one recorded, and another - saved stream
Also would be nice to see your motion.conf, which codec are you using for movies?

Looks like my issue was resolved. I did sudo apt update and sudo apt upgrade and the movies reflect what I see while streaming.

I am using an odroid xu4 running 16.04 but with latest repositories.
